src/ffi/mod.rs

```diff
@@ -162,19 +163,14 @@ fn telio_new_common(
     logger: telio_logger_cb,
     #[cfg(target_os = "android")] protect_cb: Option<telio_protect_cb>,
 ) -> telio_result {
-    // Set up `log` crate to use `logger` as 'sink' and if that works set up `tracing` to forward
-    // events to be logged via `log`.
-    if let Err(err) = log::set_boxed_logger(Box::new(logger)) {
-        eprintln!("{}", err)
-    } else if let Err(err) =
-        tracing::subscriber::set_global_default(telio_utils::tracing::TracingToLogConverter)
-    {
-        error!("{}", err);
+    let tracing_subscriber = TelioTracingSubscriber {
+        callback: logger,
+        max_level: log_level.into(),
+    };
+    if tracing::subscriber::set_global_default(tracing_subscriber).is_err() {
+        telio_log_warn!("Could not set logger, because logger had already been set by previous libtelio instance");
```

Comment:
It's the only documented error on that function. Also we need to warn the app about it, because this case is very likely due to the fact that there's no way to unset the subscriber. Some apps destroy the entire telio instance and create another one, but the subscriber will stay the same.